src/site/_data/paths/how-to-optimize-inp.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Rick and I discussed how to approach this. I originally thought that what would live at `/optimize-inp/` is what is currently at `/how-to-optimize-inp/`, but Rick and I agreed that this approach wouldn't give enough context to the guides. The content is structured this way so that guides in the collection will surface a "Also appears in: How to Optimize INP" heading below the byline to tie them all in. IMO, infra hasn't provided a way to make the grouping more clear.